src/core/audio/VorbisConsumer.cpp

```diff
@@ -53,7 +53,7 @@ auto VorbisConsumer::start(fs::path const& file) -> bool {
                 this->audioQueue.pop(std::back_inserter(buffer), buffer_size);
                 // apply gain
                 if (audioGain != 1.0) {
-                    std::for_each(begin(buffer), end(buffer), [audioGain](auto& val) { val *= audioGain; });
+                    std::for_each(begin(buffer), end(buffer), [audioGain](auto& val) { val *= static_cast<float>(audioGain); });
```

Comment:
Done

Reply:
Just an idea: Since the audiogain might be a float everywhere, especially in external libraries, we might want to introduce a type alias `using AudioGain = float;` in another PR. We might want to do this with other domain specific types too.